src/js/model/primitives/marks/Mark.js

```diff
@@ -99,7 +103,7 @@ Mark.prototype.dataset = function(id) {
     return this;
   }
 
-  this.from = undefined;
+  delete this.from;
```

Comment:
I don't think you need to delete from.  
It's already set as undefined in the Mark function.  I guess you could remove it from there and then add the property in here?